walk-in-interview/src/main/webapp/job-details/script.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {setCookie, setErrorMessage} from '../common-functions.js';
+
+const UPDATE_JOB_PATH = '../update-job/index.html';
+
+/**
+ * Gets job Id of the current job post.
+ */
+// TODO(issue/53): implement this function
+function getJobId() {
+  return 'xxxxx';
+}
+
+const submitButton = document.getElementById('update');
+submitButton.addEventListener('click', (_) => {
+
+  const jobId = getJobId();
+
+  if (jobId === '') {
+    setErrorMessage(/* errorMessageElementId= */'error-message', /* msg= */ 'Empty Job Id found.',
+      /* includesDefault= */false);
+    return;
+  }
+
+  setCookie('jobId', jobId);
+
+  fetch('../update-job/index.html', {
+    method: 'POST',
+    headers: {'Content-Type': 'application/json'},
+    credentials: 'include',
+  })
+  .then(() => {
+      /** reset the error (there might have been an error msg from earlier) */
+      setErrorMessage(/* errorMessageElementId= */'error-message', /* msg= */ '', /* includesDefault= */false);
+      window.location.href= UPDATE_JOB_PATH;
+  })
+  .catch((error) => {
+      setErrorMessage(/* errorMessageElementId= */'error-message', /* msg= */ 'Error occur when sending jobId',
```

Comment:
Is there a more user friendly message? User may not understand what we mean by 'Error occur when sending jobId'